docs/src/model_developer/market_bid_cost.md

```diff
@@ -0,0 +1,96 @@
+# PowerSystems MarketBidCost example
+
+## MarketBidCost 
+Is a Operational Cost  data structure that allows the user to run a production cost model that is very similar to most US electricity market auctions with bids for energy and ancilliary services.
+```julia
+    mutable struct MarketBidCost <: OperationalCost
+        variable::Union{Nothing, IS.TimeSeriesKey}
+        no_load::Float64
+        start_up::NamedTuple{(:hot, :warm, :cold), NTuple{3, Float64}}
+        shut_down::Float64
+        ancillary_services::Vector{Service}
+    end
+```
+
+#### Arguments
```

Comment:
It doesn't help too much replicating the struct, it is better to add a (@ref) next to the name and showcase how to use it.